apps/signing/tests/test_views.py

```diff
@@ -202,6 +216,7 @@ def test_addon_does_not_exist(self):
         assert response.data['error'] == 'Could not find add-on with id "foo".'
 
     def test_user_does_not_own_addon(self):
+        self.create_version('3.0')
```

Comment:
It just makes sure that the add-on exists. The test still fails without it but it would be a `404` instead of a `403` if the permission check was removed. Shouldn't matter but this feels right.